pkg/version/version.go

```diff
@@ -68,3 +82,27 @@ func getGoInfo() GoInfo {
 		Arch:     runtime.GOARCH,
 	}
 }
+
+func Print() {
+	fmt.Printf("replicated version %s\n", build.Version)
+
+	if build.UpdateInfo != nil {
+		fmt.Printf("Update available: %s\n", build.UpdateInfo.LatestVersion)
```

Comment:
In the examples this is also printed with structured output.  How about sending these to stderr instead?